data-lineage/src/main/java/com/wipro/ats/bdre/lineage/api/PersistenceUnit.java

```diff
@@ -70,52 +59,34 @@ public Object execute(String[] params) {
 
 	public void persist() {
 		Lineage lineage = new Lineage();
-
-//		LineageQueryInfo LineageQueryInfo = new LineageQueryInfo();
-//		LineageQueryInfo.setProcessId(122);
-//		LineageQueryInfo.setQueryId(UUID.randomUUID().toString());
-//		LineageQueryInfo.setQueryTypeId(1);
-//		LineageQueryInfo.setQueryString("select * from me");
-//		LineageQueryInfo.setInstanceExecId((long)1);
-//		LineageQueryInfo.setCreateTs(new Date());
-//		s.insert("call_procedures.InsertLineageQuery", lineageQueryInfo);
-		lineage.insertLineageQuery(lineageQueryInfo);
-
+		//inserting to LQ
 
 		for (LineageNodeInfo lineageTableNode : lineageTableNodes) {
-//			s.insert("call_procedures.InsertLineageNode", lineageTableNode);
 			lineage.insertLineageNode(lineageTableNode);
 		}
 		for (LineageNodeInfo lineageColumnNode : lineageColumnNodes) {
-//			s.insert("call_procedures.InsertLineageNode", lineageColumnNode);
 			lineage.insertLineageNode(lineageColumnNode);
 		}
 		for (LineageNodeInfo lineageFunctionNode : lineageFunctionNodes) {
-//			s.insert("call_procedures.InsertLineageNode", lineageFunctionNode);
 			lineage.insertLineageNode(lineageFunctionNode);
 		}
 		for (LineageNodeInfo lineageConstantNode : lineageConstantNodes) {
-//			s.insert("call_procedures.InsertLineageNode", lineageConstantNode);
 			lineage.insertLineageNode(lineageConstantNode);
 		}
-		for (LineageRelationInfo LineageRelationInfo : lineageRelationInfos) {
-//			s.insert("call_procedures.InsertLineageRelation", LineageRelationInfo);
-			lineage.insertLineageRelation(LineageRelationInfo);
-
+		for (LineageRelationInfo lineageRelationInfo : lineageRelationInfos) {
+			lineage.insertLineageRelation(lineageRelationInfo);
 		}
-
-		System.out.println("All nodes and relations are persisted in MySql db");
+		LOGGER.info("All nodes and relations are persisted in MySql db");
 	}
 
 	/**
 	 * Populate data beans from final node lists
 	 */
-	public void populateBeans(UniqueList<Table> finalInTableNodes, UniqueList<Table> finalOutTableNodes, List<Function> finalFunctions,UniqueList<Constant> finalConstants, List<Relation> finalRelations, String query, int processId,long instanceId) {
+	public void populateBeans(UniqueList<Table> finalInTableNodes, UniqueList<Table> finalOutTableNodes, List<Function> finalFunctions, UniqueList<Constant> finalConstants, List<Relation> finalRelations, LineageQuery lineageQuery) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this method "populateBeans" is 16 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)